photon-api/src/main/scala/com/linkedin/photon/ml/optimization/OptimizerFactory.scala

```diff
@@ -50,23 +50,21 @@ protected[ml] object OptimizerFactory {
           normalizationContext = normalizationContext,
           tolerance = config.tolerance,
           maxNumIterations = config.maximumIterations,
-          constraintMap = config.constraintMap,
           isTrackingState = isTrackingState)
 
-      case (OptimizerType.LBFGS, RegularizationType.L2 | RegularizationType.NONE) =>
+      case (OptimizerType.LBFGS, RegularizationType.L2 | RegularizationType.NONE) => {
```

Comment:
The braces (`{`) aren't necessary here, let's get rid of them to remain consistent with the other case statements.